ProjectOne/sass/layout/_header.scss

```
@@ -1,5 +1,11 @@
 @use "../abstracts" as *;
 
 .header {
-  @include center-section;
+  @include full-section;
+
+  grid-row: 1/2;
+  z-index: 2;
+  border-bottom: 1px solid $color-gray-1;
+  position: sticky;
+  top: 0;
```

các thuộc tính nằm lộn xộn thế, những thằng nào đi chung với nhau thì gom vào 1 chỗ nhé
```suggestion
  position: sticky;
  z-index: 2;
  top: 0;
  border-bottom: 1px solid $color-gray-1;
```